environments/production/inventory/groups

```diff
@@ -116,7 +116,7 @@ cluster
 [extra_packages:children]
 # Hosts to install specified additional packages on
 builder
-#cluster
+cluster
```

Comment:
Based on the discussion at https://github.com/ACRC/ansible-slurm-appliance/pull/9/changes/BASE..98d626b541bee7a2445747aed016ad9e2b540f62#r2523016757 I think this should remain commented-out?